cmd/watch_repo/cassette.tape

```diff
@@ -0,0 +1,22 @@
+Output example.gif
```

Comment:
?!?!?! What's this?

Reply:
It is for recording that cool gif using VHS